### DISCUSSION TITLE
你好之前部署了 项目是老版本 看您这边更新了 我要怎么更新

### DISCUSSION THREAD

**Discussion by shaoyouvip** [dreamhunter2333/cloudflare_temp_email#425] [General]
你好之前部署了 项目是老版本 看您这边更新了 我要怎么更新 目前分叉已经更新

**Comment by dreamhunter2333** [comment]
感谢提问，升级流程如下

https://github.com/dreamhunter2333/cloudflare_temp_email/releases/ 页面或者  https://github.com/dreamhunter2333/cloudflare_temp_email/blob/main/CHANGELOG.md 中找到当前的版本

这两个地方可以从当前版本往后的所有更改，
需要注意 Breaking Changes 是必须进行数据库 sql 执行或者 变量配置的
其他的功能更新按需配置

然后参考 https://temp-mail-docs.awsl.uk/zh/guide/cli/worker.html 和 https://temp-mail-docs.awsl.uk/zh/guide/cli/pages.html 覆盖部署之前的 worker 和 pages 即可

如果使用 UI 部署参考文档中的 UI 部署部分

**Comment by dreamhunter2333** [comment]
https://temp-mail-docs.awsl.uk/zh/guide/quick-start.html

已增加到文档
